modules/plugin/shapefile/src/test/java/org/geotools/data/shapefile/ShpFilesStreamTest.java

```diff
@@ -182,13 +182,10 @@ public void testGetReadChannelFileChannel() throws IOException {
     }
 
     @Test
-    @SuppressWarnings("PMD.UnusedVariable") // really want to just open and close
```

Comment:
Oh interesting, PMD is no longer complaining that the "read" variable is not used? Nice find. Did you just try or have a way to find unnecessary PMD annotations?

Reply:
Just trying. Seemed wrong to say the variable was unused.